python/cherrypy/server.py

```diff
@@ -0,0 +1,19 @@
+import cherrypy
+
+class Root(object):
```

Comment:
Python 3 classes are new-style by default.
```suggestion
class Root:
```

Reply:
Ok. I will change. You need to update cherrypy documentation. Just used as it is in there.